features.yaml

```
@@ -294,3 +294,31 @@ features:
       maturity: Alpha
       nextExpectedPromotion: ""
     area: Core
+  - name: "Canary Based Revision Installation"
+    link: "/docs/setup/upgrade/canary/"
+    level:
+      checklist: features/canary_based_revision_install.md
+      maturity: Beta
+      nextExpectedPromotion: ""
+    area: Core
+  - name: "Configuration Status"
+    link: "/docs/reference/config/config-status/"
+    level:
+      checklist: features/configuration_status.md
+      maturity: Beta
```

Doesn't look like all of the beta requirements are met for this one yet either. 

```suggestion
      maturity: Alpha
```